script/import_scripts/vanilla_body_parser.rb

```diff
@@ -24,6 +24,17 @@ def parse
   private
 
   def clean_up(text)
+    # <pre class="CodeBlock">...</pre>
+    text = text.gsub(/\<pre class="CodeBlock"\>(.*?)\<\/pre\>/im) { "\n```\n#{$1}\n```\n" }
```

Comment:
I see a lot of changes here that aren't tested for below. Should we add more tests to cover all these new cases?

Reply:
Agreed, will add more tests in separate PR.